### PR TITLE
DPTS-885 Logging format change

### PR DESCRIPTION
**Pull request by timrickwood**

### CODE REVIEW

File: application/utilities/common.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def valid_action(record_exists, row_data, env, invalid_action_type="false"):
             log = log + x + ":" + str(y) + ", "
```
Comment (tamaragoldschmidt) [inline application/utilities/common.py:39]: Think this should be x + "=" +

File: application/utilities/database.py [hunk 2]
```diff
@@ -61,7 +61,7 @@ def execute_db_query(db_connection, query, data, line, values, summary_count_dic
             log = log + x + ":" + str(y) + " | "
         logger.log_for_audit(
             env,
-            "action:Process row | {} | line number:{}".format(log[:-2], line),
+            "action=Process row | {} | line number={}".format(log[:-2], line),
```
Comment (tamaragoldschmidt) [inline application/utilities/database.py:64]: Line 157 DB name={} | password:secret | user:{} |

**Comment by tamaragoldschmidt** [comment]
Can you also review S3.py and secrets.py as it looks as if there are potential changes to make. _Error copying object to bucket:_

File: application/utilities/common.py (hunk 1 above)
Comment (timrickwood) [reply application/utilities/common.py:39]: yes again well spotted

File: application/utilities/common.py [hunk 3]
```diff
@@ -54,21 +54,21 @@ def archive_file(bucket, filename, event, start):
     s3_class.delete_object(bucket, filename, event, start)
     log_for_audit(
         event["env"],
-        "action:archive file:{} | bucket:{}/archive/{}".format(
+        "action=archive file:{} | bucket={}/archive/{}".format(
```
[the remark this answers is not in the document]
Comment (timrickwood) [reply application/utilities/common.py:57]: Yes well spotted - I think I went colon-blind after a while ; will also add extra pipe delimiter - eg archive | file=

File: application/utilities/database.py [hunk 4]
```diff
@@ -61,7 +61,7 @@ def execute_db_query(db_connection, query, data, line, values, summary_count_dic
             log = log + x + ":" + str(y) + " | "
```
Comment (tamaragoldschmidt) [inline application/utilities/database.py:61]: this should be =

**Review by tamaragoldschmidt — APPROVED** [review]

**Pull request merged**